etl-scripts/stored-procedures/build_flat_hiv_summary.sql

```diff
@@ -1995,10 +2009,18 @@ SELECT @person_ids_count AS 'num patients to sync';
 							when @cur_id != @prev_id then @tb_prophylaxis_duration := null
 							else @tb_prophylaxis_duration
 						end as tb_prophylaxis_duration
+
+                        case
+                            when obs regexp "!!1264=" then @tb_prophylaxis_medication := t1.value_drug
```

Comment:
is it `t1.value_drug` or `t2.value_drug`?

Reply:
should be t2.value_drug....let me correct it